src/components/subsidy-request-configuration/SubsidyRequestConfigurationContextProvider.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { createContext, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { useSubsidyRequestConfiguration } from './data/hooks';
+import EnterpriseAppSkeleton from '../EnterpriseApp/EnterpriseAppSkeleton';
+
+export const SubsidyRequestConfigurationContext = createContext();
+
+const SubsidyRequestConfigurationContextProvider = ({ enterpriseUUID, children }) => {
+  const { subsidyRequestConfiguration, isLoading } = useSubsidyRequestConfiguration(enterpriseUUID);
+
+  const context = useMemo(() => ({
+    subsidyRequestConfiguration,
+  }), [subsidyRequestConfiguration, isLoading]);
```

Comment:
nit: probably don't need `isLoading` in the dependency array here since it's not used by the `useMemo` function.